### PR TITLE
chore: update Dependabot configuration

### PR DESCRIPTION
**Pull request by merlinnot**

### CODE REVIEW

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -5,12 +5,22 @@ updates:
...
-        versions:
-          - '>= 13'
+    open-pull-requests-limit: 99
+    package-ecosystem: github-actions
+    reviewers:
+      - 'ridedott/platform'
+    schedule:
+      interval: daily
+  - allow:
+      - dependency-type: direct
```
Comment (mad-it) [inline .github/dependabot.yaml:15]: is this not already present on line 2-3 ?

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -5,12 +5,22 @@ updates:
...
+    commit-message:
+      include: scope
+      prefix: chore
+    directory: /
```
Comment (mad-it) [inline .github/dependabot.yaml:19]: Is this not already present on line 7?

File: .github/dependabot.yaml (hunk 1 above)
Comment (merlinnot) [reply .github/dependabot.yaml:15]: This is a separate entry in an array, a separate configuration object.

File: .github/dependabot.yaml (hunk 2 above)
Comment (merlinnot) [reply .github/dependabot.yaml:19]: Same as above.

File: .github/dependabot.yaml (hunk 1 above)
Comment (mad-it) [reply .github/dependabot.yaml:15]: Ahh yeah, i was unsure. Indenting hard to read in these diffs 🙈

**Review by mad-it — APPROVED** [review]
My approval has no weight, but you get it anyways 😂

**Review by acazacu — APPROVED** [review]

**Pull request merged**